metabox/metabox/core/runner.py

```diff
@@ -276,9 +276,12 @@ def run(self):
             if not scn.has_passed():
                 self.failed = True
                 logger.error(scenario_description + " scenario has failed.")
-                logger.error(
-                    "Scenario output:\n" + scn.get_output_streams().strip()
-                )
+
+                # let's strip < from the output to avoid confusing loguru
```

Comment:
Looks like the comment does not match what's being done, cause in the code you escape `<` instead of stripping it from the output.